nucypher/control/controllers.py

```diff
@@ -310,14 +310,25 @@ def handle_request(self, method_name, control_request, *args, **kwargs) -> Respo
                            self.emitter.MethodNotFound)
 
         try:
+            # handle request headers
+            request_headers = control_request.headers
+            request_headers = {header.upper(): value for (header, value) in request_headers.items()}
```

Comment:
Heh, the headers must begin with an uppercase letter?

Also, just a style comment, parenthesis are not needed here:
```suggestion
            request_headers = {header.upper(): value for header, value in request_headers.items()}
```